src/main/java/com/facebook/openwifirrm/Launcher.java

```diff
@@ -82,21 +82,24 @@ private RRMConfig readRRMConfig(File configFile) throws IOException {
 			// Read file
 			logger.info("Reading config file '{}'", configFile.getPath());
 			String contents = Utils.readFile(configFile);
-			JSONObject userConfig = new JSONObject(contents);
-
```

Comment:
please leave in these blank lines for spacing (it was intended to create "paragraphs" of comments+code)